orion-server/src/main/java/com/pinterest/orion/core/actions/aws/ReplaceEC2InstanceAction.java

```diff
@@ -71,6 +74,9 @@ public void initialize(Map<String, Object> config) throws PluginConfigurationExc
       throw new PluginConfigurationException(
           "Cannot find key " + Ec2Utils.CONF_ROUTE53_ZONE_NAME + " in config " + getName());
     }
+    if (config.containsKey(ATTR_FALLBACK_MAPPING)) {
+      fallbacksMap = (Map<String, List<String>>)config.get(ATTR_FALLBACK_MAPPING);
```

Comment:
should make this an empty map if the config doesn't contain the key, or add null checks everywhere else to prevent NPE